src/axom/quest/DistributedClosestPoint.hpp

```diff
@@ -412,23 +419,53 @@ class DistributedClosestPointImpl
 
 public:
   /**
-   * Utility function to set the array of points
+   * Import object mesh points from the object blueprint mesh into internal memory.
    *
-   * \param [in] coords The root group of a mesh blueprint's coordinate values
+   * \param [in] mdMeshNode The blueprint mesh containing the object points.
+   * \param [in] valuesPath The path to the mesh points.
    * \note This function currently supports mesh blueprints with the "point" topology
+   *
+   * TODO: See if some of the copies in this method can be optimized out.
```

Comment:
Minor: This `TODO` comment is more relevant to developers than to users.
Perhaps it (and possibly others) should be moved inside the function so it's not part of the user docs.